REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.5
+julia 0.6.0-pre.alpha.0
```

Comment:
Update: now that the package is compatible with Julia 0.5, this is fixed as well.